src/cljs/main/broadfcui/page/workspace/analysis/igv.cljs

```diff
@@ -10,61 +11,58 @@
    [broadfcui.utils.user :as user]
    ))
 
-(defonce ^:private igv-styles-loaded? (atom false))
-
-(defn- load-igv-styles []
-  (.. (js/$ "head") ; IGV uses jQuery, so I'm using it here too.
-      (append "<link rel=\"stylesheet\" type=\"text/css\" href=\"https://igv.org/web/release/1.0.1/igv-1.0.1.css\">"))
-  (reset! igv-styles-loaded? true))
-
 (defn- options [tracks token]
   (clj->js
-   {:genome "hg19"
-    :trackDefaults {:palette ["#00A0B0" "#6A4A3C" "#CC333F" "#EB6841"]
-                    :bam {:coverageThreshold 0.2
-                          :coverageQualityWeight true}}
+   {
+    ;; TODO: should we set hg38 as the default?
+    :genome "hg19"
+    :oauthToken token
```

Comment:
IGV 2.x supports setting an oauth token at the root of IGV instead of per-track. This would even allow us to use tracks pointing at GA4GH variant stores or htsget to google servers in the future.